BiasAndFairnessServers/src/models/DeepEvalProject.py

```
@@ -0,0 +1,70 @@
+    
+    def __init__(
+        self,
+        id: str,
+        name: str,
+        description: str,
+        model: Dict[str, Any],
+        dataset: Dict[str, Any],
+        metrics: Dict[str, bool],
+        metric_thresholds: Dict[str, float],
+        tenant: str,
+        created_at: datetime = None,
```

Ruff (0.14.4)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)